src/controllers/users.js

```diff
@@ -32,6 +32,13 @@ userControllers.update = (req, res) => {
   // receives updated json for user in body
   // amends db record
   // status codes: 200 (success), 400 (bad request)
+  const id = req.params.id
+  User.findByIdAndUpdate(id, req.body, { new: true })
+    .then(updatedUser => res.send(updatedUser))
+    .catch(err => {
+      const errorObj = { message: `Database error: ${err.message}` }
+      res.status(400).send(errorObj)
```

Comment:
Hmm okay, it seems findByIdAndUpdate returns an error if user does not exist, in this case should be 404, in other cases prop 500.

https://en.wikipedia.org/wiki/List_of_HTTP_status_codes